### PR TITLE
fix: Add create event to ShibbolethController and Add missing member …

### PR DESCRIPTION
**Pull request by DanielPalafox**
…name

### CODE REVIEW

**Comment by DanielPalafox** [comment]
[[QA] Registry event captured for OAuth new registration needs fixing](https://trello.com/c/jbdfmzfE/8894-qa-registry-event-captured-for-oauth-new-registration-needs-fixing)

**Comment by DanielPalafox** [comment]
[[QA] Fix registry events label for OAuth](https://trello.com/c/VLUzrXap/8915-qa-fix-registry-events-label-for-oauth)

**Comment by DanielPalafox** [comment]
[[QA] Issues capturing institutional and social signin events within the registry](https://trello.com/c/pJsqXRei/8893-qa-issues-capturing-institutional-and-social-signin-events-within-the-registry)

File: orcid-web/src/main/java/org/orcid/frontend/web/controllers/RegistrationController.java [hunk 1]
```diff
@@ -236,19 +236,19 @@ public void validateGrcaptcha(HttpServletRequest request, @RequestBody Registrat
...
-            // the login page
-            if (request.getSession().getAttribute(GRECAPTCHA_SESSION_ATTRIBUTE_NAME) == null || PojoUtil.isEmpty(reg.getGrecaptcha())
-                    || !reg.getGrecaptcha().getValue().equals(request.getSession().getAttribute(GRECAPTCHA_SESSION_ATTRIBUTE_NAME))) {
-                r.setUrl(getBaseUri() + "/register");
-                return r;
-            }
-
-            usedCaptcha = true;
-        }
+        // If recatcha wasn't loaded do nothi
```
Comment (amontenegro) [inline orcid-web/src/main/java/org/orcid/frontend/web/controllers/RegistrationController.java:239]: @DanielPalafox why did you commented this code?

**Review by amontenegro — CHANGES_REQUESTED** [review]

File: orcid-web/src/main/java/org/orcid/frontend/web/controllers/RegistrationController.java (hunk 1 above)
Comment (DanielPalafox) [reply orcid-web/src/main/java/org/orcid/frontend/web/controllers/RegistrationController.java]: @amontenegro Oh my mistake. Done

**Review by amontenegro — APPROVED** [review]

**Pull request merged**